src/components/cards/LargeHorizontalCard/largeHorizontalCard.css

```diff
@@ -0,0 +1,34 @@
+.videolib-hr-largecard-imgresize {
```

Comment:
Suggestion: here "videolib" is not necessary, it can help you shorten the class names